src/framework/preferences/css/SeparatedPanelPrefsEditorFrame.css

```diff
@@ -1 +1 @@
\ No newline at end of file
+@font-face{font-family:"OpenSans";src:url("../../../lib/fonts/OpenSans-Light.ttf");font-weight:300;font-style:normal}@font-face{font-family:'OpenSans';src:url("../../../lib/fonts/OpenSans-Regular.ttf");font-weight:normal;font-style:normal}@font-face{font-family:'OpenSans';src:url("../../../lib/fonts/OpenSans-Semibold.ttf");font-weight:600;font-style:normal}html{overflow:visible}.fl-prefsEditor-separatedPanel{background-color:#fff;background-image:url("../images/default/separatedpanelbg.png");background-repeat:repeat;height:100%;overflow-x:auto;overflow-y:hidden;color:#766d68;padding:1px 0;margin:0;}.fl-prefsEditor-separatedPanel .fl-prefsEditor{font-family:'OpenSans',"Myriad Pro",Helvetica,Arial,sans-serif;line-height:1.2em !important}.fl-prefsEditor-separatedPanel .fl-prefsEditor-panels{white-space:nowrap;display:table;border-spacing:20px 0;}.fl-prefsEditor-separatedPanel .fl-prefsEditor-panels .fl-prefsEditor-panel{border:1px solid #615e59;border-radius:5px;box-shadow:2px 2px #7a766d;display:inline-block;padding:1em;min-width:25em;background-color:#fff;white-space:normal;display:table-cell}.fl-prefsEditor-separatedPanel h2{color:#766d68;padding:0 0 .5em 0;margin:0 0 1.5em 0;width:100%;border-bottom:solid #ccc 1px;box-shadow:0 6px 7px -6px rgba(0,0,0,0.3);font-size:1.2em;text-transform:lowercase;line-height:1.7em;height:2em}.fl-prefsEditor-separatedPanel .fl-inputs{float:none}.fl-prefsEditor-separatedPanel label{font-size:1em}.fl-prefsEditor-separatedPanel .fl-prefsEditor-separatedPanel-toc{width:6em;height:2em;margin-right:10px;float:left;}.fl-prefsEditor-separatedPanel .fl-prefsEditor-separatedPanel-toc label{line-height:1.7em !important}.fl-prefsEditor-separatedPanel .fl-links-buttons-options label{width:21em;display:inline-block}.fl-prefsEditor-separatedPanel .fl-links-buttons-options li{margin-bottom:.4em}.fl-prefsEditor-separatedPanel h2 label,.fl-prefsEditor-separatedPanel h2 .heading-text{float:left;padding-top:.2em;font-size:1.3em;font-weight:600}.fl-prefsEditor-separatedPanel .fl-icon-next,.fl-prefsEditor-separatedPanel .fl-icon-prev{font-family:'InfusionIcons' !important;speak:none;font-style:normal;font-weight:normal;font-variant:normal;text-transform:none;-webkit-font-smoothing:antialiased;float:left;margin:.2em .3em 0 0;font-size:1.5em}.fl-prefsEditor-separatedPanel .fl-icon-next:before{content:"\e001"}.fl-prefsEditor-separatedPanel .fl-icon-prev:before{content:"\e002"}.fl-prefsEditor-separatedPanel.fl-theme-bw{background-image:none}.fl-prefsEditor-separatedPanel.fl-theme-wb{background-image:none}.fl-prefsEditor-separatedPanel.fl-theme-yb{background-image:none}.fl-prefsEditor-separatedPanel.fl-theme-by{background-image:none}.fl-prefsEditor-separatedPanel.fl-theme-lgdg{background-image:none}
```

Comment:
That's correct. All css files in src/framework/preferences/css/ directory should not be committed. They are dynamically generated at running grunt build script.